definitions/divorce/json/CaseEvent.json

```diff
@@ -2197,5 +2197,19 @@
     "RetriesTimeoutURLAboutToSubmitEvent": "120,120",
     "SecurityClassification": "Public",
     "ShowSummary": "Y"
+  },
+  {
+    "LiveFrom": "25/06/2019",
+    "CaseTypeID": "DIVORCE",
+    "ID": "IssueAOSOffline_CoRespondent",
+    "Name": "Issue AOS offline - Co-Respondent",
+    "Description": "Issue AOS (offline) for Co-Respondent",
+    "DisplayOrder": 1,
+    "PreConditionState(s)": "*",
+    "PostConditionState": "*",
```

Comment:
By doing this we will have this event showing up at every stage of the application. Are we sure we want this? Is this how is should be?

Reply:
Also, you'll see this option regardless of the reason for divorce being Adultery or not

Reply:
Good points. Did we talk about a few specific pre-conditions last week @iichr ? Does your story say something about that?

Reply:
This is all that's in the story: 
> Pre-condition and post-condition states are no longer relevant for the co-respondent.
So not quite clear, open to interpretation

Reply:
Amended per the discussion last week, pre-conditions can be aosStarted and aosAwaiting